docs/release-notes/8.5.md

```diff
@@ -0,0 +1,212 @@
+# EuroLinux 8.5 Release Notes
+
+EuroLinux 8.5 is the first minor release created in a continuous way. It was
+also the first version when we deployed our engineer to remote sites (in most
+cases virtual) to provide support for organisations that rebuild RHEL/EuroLinux
+from sources with Gaia build system. This version code name is Tirana - the
+capital city of Albania.
+
+
+Below you can find the essential changes in EuroLinux 8.5 release for `x86_64`
+architecture. The `aarch64` GA release is slightly delayed. Release notes for
+`aarch64` will be provided in a separate document.
+
+## Dynamic programming languages, Web and Database servers
+
+Later versions of the following components are now available:
+
+-   **Ruby 3.0** - module `ruby` stream 3.0.
+-   **nginx 1.20** - module `nginx` stream 1.20
+-   **Nodejs 16** - module `nodejs` stream 16
+
+## Compiler Toolset
+
+The following compiler toolsets have been updated in EL 8.5:
+
+-   **GCC Toolset 11** - software collection `gcc-toolset-11`
+-   **LLVM Toolset 12.0.1** - module `llvm-toolset` stream `rhel`
+-   **Rust Toolset 1.54.0** - module `rust-toolset` stream `rhel`
+-   **Go Toolset 1.16.7** - module `go-toolset` stream `rhel`
+
+## Security
+
+Most of the security features are connected to rebased/updated versions
+of the packages. The most crucial security changes include:
+
+- rsyslog - now support OpenSSL
+- OpenSCAP framework - added new profiles and multiple changes and enhancement
+- cryptopolicies were updated - these system-wide cryptographic policies from this
+  version support different policies for different backends (scoped policies)
+
+## High Availability and Resilient Storage
+
+- cmirror was updated to version 2.03.12
+- corosync was updated to version 3.1.5
+- pacemaker was updated to version 2.1.0
+- resource-agents-paf package was added to HA an RS repositories
+
+## General updates and improvements
+
+Among the others, the following packages were updated and enhancement:
```

Comment:
"and enhancement"?
Maybe _and enhanced_ if we're talking about the quality improvements these updates bring.